src/ServiceControl.Config/Xaml/Behaviours/RichTextBoxHelper.cs

```diff
@@ -78,7 +78,8 @@ public static void SetDocumentXaml(DependencyObject obj, string value)
                         // When the document changes update the source
                         richTextBox.TextChanged += (obj2, e2) =>
                         {
-                            if (obj2 is RichTextBox richTextBox2)
+                            var richTextBox2 = (RichTextBox)obj2;
+                            if (richTextBox2 != null)
```

Comment:
Code inspection. It reads better: 

> Is `obj2` a `RichTextBox`

Is easier to read than assigning it and checking for null, which is essentially answering the same question.

Reply:
Hmm. that is weird because we switched that to the new code styles which allow pattern matching. The above code is the pattern matching variant which is recommended. We added that code when we move SC codebase to the new code styles. Could it be that you applied that recommendation with the custom dotsettings file per csproj?

Reply:
Yeah, something seems off if it wanted you to move away from the newer C# style.